CMakeLists.txt

```diff
@@ -36,8 +48,8 @@ cmake_dependent_option(KR_VELOC_BAREBONE "Use the barebone branch of VeloC" OFF
 
 # VeloC backend
 if (KR_ENABLE_VELOC)
-  find_package(VeloC REQUIRED)
-  target_link_libraries(resilience PUBLIC veloc::veloc)
+  find_package(veloc REQUIRED)
+  target_link_libraries(resilience PUBLIC veloc::client)
```

Comment:
Probably should propose these VeloC changes as part as another PR

Reply:
VeloC is changing the way their cmake works (writing a config file). It's a bit messy but maybe the PR should be titled "Update code to work with newer versions of Kokkos and VeloC develop"